ops/app.yml

```diff
@@ -4,4 +4,5 @@
 
   roles:
     - cgus-data
```

Comment:
Why not call this one `cgus-data-versions` if a new `cgus-data-snapshots` is created?

Reply:
Because I prefer considering that `cgus-data` is the real database and the snapshot database is only a backup to recover from filtering errors.

Reply:
What about we name both `cgus-versions` and `cgus-snapshots`? 🙂

Reply:
okay 👍